.github/workflows/copyright-validate.yaml

```diff
@@ -0,0 +1,19 @@
+name: Check Copyright Headers
+
+on:
+  # - pull_request [should we include for all types]
+  push:
+    branches:
+      - gh-copyright-workflow
+jobs:
+  copywrite:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+
+      - name: Install copywrite
```

Comment:
None blocking comment:
I will suggest just to define `uses` for checkout + copywrite, as [we have in other jobs](https://github.com/hashicorp/boundary-ui/blob/main/.github/workflows/monorepo-validate.yaml#L27-L28), and just use `name` for check header compliance.
I do suggest this approach because we do not perform operations within checkout action or install copywrite, we define the action.

In example:
```
- uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
- uses: hashicorp/setup-copywrite@32638da2d4e81d56a0764aa1547882fc4d209636 # v1.1.3
- name: Check Header Compliance
        run: copywrite headers --plan

Reply:
yeah that's fair, I'll remove the labels